Dockerfile

```diff
@@ -46,14 +60,14 @@ ARG GIT_COMMIT=
 ENV VERSION ${GIT_TAG}
 ENV GIT_COMMIT ${GIT_COMMIT}
 
-RUN python setup.py develop
-
 CMD ["python", "manage.py", "--help"]
 
 ### Dist
 FROM app AS dist
 
+RUN $POETRY_HOME/bin/poetry install --compile --only deploy
+
 ### Dev
 FROM app AS dev
 
-RUN pip install --no-cache-dir -c /code/constraints.txt -r /code/dev-requirements.txt
+RUN $POETRY_HOME/bin/poetry install --compile --only dev
```

Comment:
Looks good. Docker build had a few nitpicks, but these aren't dealbreakers:

```
 3 warnings found (use docker --debug to expand):                                                                                                                                                           
 - FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 1)
 - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 60)
 - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 61)
```